models/users.js

```diff
@@ -31,6 +31,7 @@ const addOrUpdate = async (userData, userId = null) => {
         await userModel.doc(userId).set({
           ...user.data(),
           ...userData,
+          data,
```

Comment:
Either Destructure the `data` here or get `active-hours` and `status` as part of `userData` object.

```js
        await userModel.doc(userId).set({
          ...user.data(),
          ...userData,
          ...data,
          });
          
```